test/lit/wasm-split/module-names.wast

```diff
@@ -16,7 +16,7 @@
 
 (module
   (func $foo
-    (nop)
+    (call $bar)
```

Comment:
Why change this?

Reply:
To prevent `$bar` from being removed for being unused in the secondary module.